### PR TITLE
add exception handling to DfSchemaHistory and adjust diffmodel.vm

### PR DESCRIPTION
**Pull request by Ryo-chin**

### CODE REVIEW

File: dbflute-engine/embedded/templates/doc/html/diffmodel.vm [hunk 1]
```diff
@@ -16,961 +16,968 @@
 <!DOCTYPE html>
 <html lang="en"> <!-- main language (to avoid HTML validator) -->
 <head>
-	<meta charset="UTF-8" />
-	<title>${database.projectName} ${selector.diffModelSmallTitle}</title>
```
Comment (dbflute) [inline dbflute-engine/embedded/templates/doc/html/diffmodel.vm]: Big difference exists in the file.
Is it by auto-formatter?
Please don't change the format of unrelated lines.

File: dbflute-engine/src/main/java/org/dbflute/logic/doc/historyhtml/DfSchemaHistory.java [hunk 2]
```diff
@@ -205,10 +205,20 @@ protected void assertDiffElementMap(String key, Object value) {
...
-        // TODO hakiba add exception handling  (2018/04/28)
-        List<DfHacoMapPiece> pieceList = hacoMapFile.readPieceList(clientDirPath);
-        OptionalThing<DfHacoMapPickup> optPickup = hacoMapFile.readPickup(clientDirPath);
-        _hacoMapPickup = hacoMapFile.merge(optPickup, pieceList);
+        // TODO done hakiba add exception handling  (2018/04/28)
+        try {
+            List<DfHacoMapPiece> pieceList = hacoMapFile.readPieceList(clientDirPath);
+            OptionalThing<DfHacoMapPickup> optPickup = hacoMapFile.readPickup(clientDirPath);
+            _hacoMapPickup = hacoMapFile.merge(optPickup, pieceList);
+        } catch (RuntimeException e){
```
Comment (dbflute) [inline dbflute-engine/src/main/java/org/dbflute/logic/doc/historyhtml/DfSchemaHistory.java]: What is this exception handling for?
If you want to improve debug information,
e.g. you should show "clientDIrPath" in the exception message.

While, the addItem("Exception") is unneeded because IllegalStateException wraps it.
(Sometimes it can do for emphasis, but I think it does not mean here)

Comment (Ryo-chin) [reply dbflute-engine/src/main/java/org/dbflute/logic/doc/historyhtml/DfSchemaHistory.java]: Since clientDirPath is a fixed value, I decided that log output is not necessary. However, FilePath may be changed, so I will log it.

File: dbflute-engine/embedded/templates/doc/html/diffmodel.vm (hunk 1 above)
Comment (Ryo-chin) [reply dbflute-engine/embedded/templates/doc/html/diffmodel.vm]: sorry, I fixed it.

**Comment by dbflute** [comment]
Perfect! Thanks.
